modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1787,6 +1742,15 @@ public ObjectEntry moveObjectEntryToTrash(
 		return objectEntry;
 	}
 
+	@Override
+	public void moveRelatedObjectEntriesToTrash(
```

Comment:
Can we keep just the current `deleteRelatedObjectEntries` method and change only the internal logic?
Something like we did in [DefaultObjectEntryManagerImpl#_deleteObjectEntry](https://github.com/liferay/liferay-portal/blob/67f4a21da5673160e872ba57258d215c931565af/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java#L1497-L1520).
Once we have the configuration created, my idea is to check a new private method in the objectDefinition with the name `ObjectDefinitionModelImpl#isEnableRecycleBin`, for now you can check the FF and the status of the parent entry in the relationship.